backend/apps/github/api/user.py

```
@@ -1,60 +1,49 @@
-            Response: The serialized user data or a 404 error if the user is not found.
-
-        """
-        try:
-            user = User.objects.get(login=login)
-            serializer = self.get_serializer(user)
-            data = serializer.data
-            return Response(data)
-        except User.DoesNotExist:
-            return Response({"detail": "User not found."}, status=404)
+@router.get("/{login}", response=UserSchema)
+def get_user(request: HttpRequest, login: str) -> UserSchema | None:
```

Ruff (0.11.9)

Redefinition of unused `get_user` from line 38

(F811)